setup.cfg

```diff
@@ -11,10 +11,17 @@ install_requires =
     uvicorn[standard]==0.29.0
     opensearch-py==2.5.0
     boto3==1.34.85
+    beautifulsoup4==4.12.3
```

Comment:
I can't put this comment exactly where I want, but after line 8 we should add `raise_search.commands` as a subpackage.